BAZProjectUnitTests/TrendingMoviesViewControllerTests.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  TrendingMoviesViewControllerTests.swift
+//  BAZProjectUnitTests
+//
+//  Created by gescarcega on 30/03/23.
+//
+
+import XCTest
+@testable import BAZProject
+
+final class TrendingMoviesViewControllerTests: XCTestCase {
+    
+    var sut: TrendingMoviesViewController!
+    var viewModel: TrendingMoviesViewModelMock!
+
+    override func setUp() {
+        super.setUp()
+        viewModel = TrendingMoviesViewModelMock()
+        sut = TrendingMoviesViewController.getInstance(with: viewModel)
+    }
+
+    override func tearDown() {
+        sut = nil
+        super.tearDown()
+    }
+
+    func test_NumberOfRowsInSection_ShouldBeThree() {
+        //Given
+        let movies = [Movie(title: "title1"), Movie(title: "title2"), Movie(title: "title3")]
+        
+        //When
+        viewModel.movies = movies
+        sut.viewDidLoad()
```

Comment:
Podemos usar `loadViewIfNeeded` como la forma recomendada para cargar vistas en pruebas, este llamará `viewDidLoad`
```suggestion
        sut.loadViewIfNeeded()
```